KarhooUISDK/Screens/QuoteList/QuoteListFilters/Filters/QuoteTypeFilter.swift

```diff
@@ -17,7 +17,12 @@ extension QuoteListFilters {
         var filterCategory: Category { .quoteTypes}
         
         var localizedString: String {
-            rawValue
+            switch self {
+            case .fixed:
+                return UITexts.Generic.fixed
+            case .metered:
+                return UITexts.Generic.metered
+            }
         }
 
         func conditionMet(for quote: Quote) -> Bool {
```

Comment:
We do not have any other option for now, since data in Quote model are also saved as raw string. Once we'll change it, we can easily change the comparison details.

Reply:
what about something like: 
`switch self:
 case .fixed:
    return quote.quoteType == .fixed
 case .metered:
    return quote.quoteType == .metered`

I agree, that this solution is less developer-friendly, but at least will not create issue while renaming.
Btw. Shoudn't be `QuotyType.estimated` assigned to one of QuoteListFilters types?